packages/dev/core/src/Materials/PBR/pbrBaseMaterial.ts

```diff
@@ -1733,6 +1737,10 @@ export abstract class PBRBaseMaterial extends PushMaterial {
                             }
                         }
                     }
+
+                    if (this._useAlternateRoughRadiance) {
+                        defines.ALTERNATE_ROUGH_RADIANCE = true;
+                    }

```

Comment:
See my other comment, should be:

```suggestion
                    defines.ALTERNATE_ROUGH_RADIANCE = this._useAlternateRoughRadiance;
```